test/WebSites/ControllersFromServicesWebSite/Startup.cs

```diff
@@ -3,14 +3,9 @@
 
 using System;
 using System.Reflection;
+using ControllersFromServicesClassLibrary;
 using Microsoft.AspNet.Builder;
 using Microsoft.Framework.DependencyInjection;
-using ControllersFromServicesClassLibrary;
```

Comment:
We already have coverage for DI intergration with Autofac as part of AutofacWebsite. Hence removing from here rather than converting it to the official package.